includes/namespace.php

```diff
@@ -80,6 +82,59 @@ function deactivate( $network_wide ) {
 register_activation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\activate' );
 register_deactivation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\deactivate' );
 
+$heading = __( 'Web Stories plugin could not be initialized.', 'web-stories' );
+$body    = sprintf(
+	/* translators: %s: build commands. */
+	__( 'You appear to be running an incomplete version of the plugin. Please run %s to finish installation.', 'web-stories' ),
+	'<code>composer install &amp;&amp; npm install &amp;&amp; npm run build</code>'
+);
+
+if (
+	! class_exists( '\Google\Web_Stories\Plugin' ) ||
+	! file_exists( WEBSTORIES_PLUGIN_DIR_PATH . '/assets/js/edit-story.js' )
+) {
+	/**
+	 * Displays an admin notice about why the plugin is unable to load.
+	 *
+	 * @return void
+	 */
+	function _print_missing_build_admin_notice() {
+		global $heading, $body;
+		?>
+		<div class="notice notice-error">
+			<p>
+				<strong><?php echo esc_html( $heading ); ?></strong>
+			</p>
+			<p>
+				<?php echo wp_kses_post( $body ); ?>
+			</p>
+		</div>
+		<?php
+
+		// Don't pollute global space.
+		unset( $heading, $body );
+	}
+
+	add_action( 'admin_notices', __NAMESPACE__ . '\_print_missing_build_admin_notice' );
+}
+
+// In CLI context, existence of the JS files is not required.
+if (
+	! class_exists( '\Google\Web_Stories\Plugin' ) &&
```

Comment:
Where there a time you be running WordPress is php cli environment and would not be using WP CLI. Seems like a use case we shouldn't support...

Reply:
When running `npm run test:php` for example

Reply:
Doesn't that mean you have to build php and js in php unit tests. Wasn't this something you wanted to avoid?

Reply:
No.

This change means that if you are running tests, or running WP-CLI, you only get a warning if you forgot  `composer install`. Missing JS files don't matter in this context.

Reply:
This is in the message `composer install && npm install && npm run build`? That is saying to build the js as well. Am I missing something?